docs/src/pages/index.js

```diff
@@ -328,9 +328,12 @@ export default function IndexPage() {
                 <h1 className="text01">
                   <span>Get WhatsApp</span>
                   <br></br>
-                  <span>notification when your</span>
+                  <span>notification when</span>
+                  <br></br>
+                  <span>
+                    your <span className="text05">web app is down</span>
+                  </span>
```

Comment:
Can we not do this? Keep the text plain and continuous without span and br, except for the `web app is down` text. To change the line height, I think you can use the `leading-x` class names from tailwind. Check out the doc [here](https://tailwindcss.com/docs/line-height).